index-config/src/main/solr-config/uniprot-collections/uniref/conf/schema.xml

```diff
@@ -19,8 +19,10 @@
 		required="false" multiValued="false" />
 	<field name="length" type="int" indexed="true" stored="false"
 		required="false" multiValued="false" />
-	<field name="created" type="date" indexed="true" stored="false"
-		required="false" docValues="true" />
+	<field name="created_sort" type="date" indexed="true"
+		stored="false" required="false" docValues="true" />
+	<field name="created" type="rdates" indexed="true" stored="false"
+		required="false" />
 	<field name="uniprot_id" type="basic_ci" indexed="true"
```

Comment:
Why do we need this created date change?

Reply:
This way we can make the range search works